ui/components/src/StatChart/StatChart.test.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { PersesThemeProvider } from '../context/PersesThemeProvider';
+import { UnitOptions } from '../model';
+import { StatChart, StatChartData } from './StatChart';
+
+describe('StatChart', () => {
```

Comment:
Yeah I am, I'll do GaugeChart and LineChart in a follow up PR, they're a little harder to write than StatChart, but Luke's [recent PR](https://github.com/perses/perses/pull/467) helps a ton, also going to get rid of the swc changes in this PR since it won't work with project references.